libraries/standard/mqtt/integration-test/test_config.h

```diff
@@ -0,0 +1,100 @@
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#ifndef TEST_CONFIG_H
+#define TEST_CONFIG_H
+
+/**************************************************/
+/******* DO NOT CHANGE the following order ********/
+/**************************************************/
+
+/* Include logging header files and define logging macros in the following order:
+ * 1. Include the header file "logging_levels.h".
+ * 2. Define the LIBRARY_LOG_NAME and LIBRARY_LOG_LEVEL macros depending on
+ * the logging configuration for DEMO.
+ * 3. Include the header file "logging_stack.h", if logging is enabled for DEMO.
+ */
+
+#include "logging_levels.h"
+
+/* Logging configuration for the Demo. */
+#ifndef LIBRARY_LOG_NAME
+    #define LIBRARY_LOG_NAME    "TEST"
+#endif
+
+#ifndef LIBRARY_LOG_LEVEL
+    #define LIBRARY_LOG_LEVEL    LOG_INFO
+#endif
+#include "logging_stack.h"
+
+/************ End of logging configuration ****************/
+
+/**
+ * @brief MQTT server host name.
+ *
+ * This test uses the Mosquitto test server. This is a public MQTT server; do not
+ * publish anything sensitive to this server.
+ * Mosquitto MQTT broker can run locally as an alternate option. Please refer to
+ * the instructions in https://mosquitto.org/ for running a Mosquitto broker
+ * locally.
+ */
+#ifndef BROKER_ENDPOINT
+    #define BROKER_ENDPOINT    "test.mosquitto.org"
+#endif
+
+/**
+ * @brief Length of MQTT server host name.
+ */
+#ifndef BROKER_ENDPOINT_LENGTH
+    #define BROKER_ENDPOINT_LENGTH    ( ( uint16_t ) ( sizeof( BROKER_ENDPOINT ) - 1 ) )
+#endif
+
+/**
+ * @brief MQTT server port number.
+ *
+ * In general, port 8883 is for secured MQTT connections.
+ */
+#define BROKER_PORT    ( 8883 )
+
+/**
+ * @brief Path of the file containing the server's root CA certificate.
+ *
+ * This certificate should be PEM-encoded.
+ */
+#ifndef SERVER_ROOT_CA_CERT_PATH
+    #define SERVER_ROOT_CA_CERT_PATH    "/home/ubuntu/Repos/aws-iot-device-sdk-embedded-C/libraries/standard/mqtt/integration-test/mosquitto.org.crt"
+#endif
+
+/**
+ * @brief Length of path to server certificate.
+ */
+#define SERVER_CERT_PATH_LENGTH    ( ( uint16_t ) ( sizeof( SERVER_CERT_PATH ) - 1 ) )
```

Comment:
SERVER_CERT_PATH  or SERVER_ROOT_CA_CERT_PATH?

Reply:
`SERVER_CERT_PATH` is not used in the test, so have removed it.